tools/test-ci.sh

```diff
@@ -0,0 +1,3 @@
+pushd `dirname $0`/.. > /dev/null
```

Comment:
Please add a shebang line at the top of this file.

Also, just use `cd`, `pushd` isn’t a common feature of all POSIX shells (and there’s no need to do the redirect).